File: cpp/Komentarz.cpp
#include "Komentarz.h"

string Komentarz::komentarz = "";

void Komentarz::wyswietlKomentarze()
{
    if (komentarz == "") return;
    else cout << komentarz;
}

void Komentarz::dodajKomentarz(string nowyKomentarz) {
    komentarz += nowyKomentarz + "\n";
}

void Komentarz::czyscKomentarze()
{
    komentarz = "";
}

File: java/pl/edu/pg/po/projekt2/Komentarz.java
package pl.edu.pg.po.projekt2;

public class Komentarz {
    private static String komentarz = "";

    public static String wyswietlKomentarze() {
        return komentarz;
    }

    public static void dodajKomentarz(String nowyKomentarz) {
        komentarz += nowyKomentarz + "\n";
    }

    public static void czyscKomentarze() {
        komentarz = "";
    }
}
